docs/windows.md

```diff
@@ -125,127 +157,24 @@ document.
 
-#### Add Windows kube-proxy DaemonSet
-
-Add Windows-compatible versions of kube-proxy by applying file `kube-proxy.yaml`.
-
-Download `kube-proxy.yaml` from kubernetes official repository and set
-kube-proxy version.
-
-```bash
-# Example:
-curl -L "https://github.com/kubernetes-sigs/sig-windows-tools/releases/download/v0.1.5/kube-proxy.yml" | sed 's/VERSION/v1.23.5/g' > kube-proxy.yml
-```
-
-Replace the content of `run-script.ps1` in configmap named `kube-proxy-windows`
-with following:
-
-```yaml
-apiVersion: v1
-data:
-  run-script.ps1: |-
-    $ErrorActionPreference = "Stop";
-    mkdir -force /host/var/lib/kube-proxy/var/run/secrets/kubernetes.io/serviceaccount
-    mkdir -force /host/k/kube-proxy
-
-    cp -force /k/kube-proxy/* /host/k/kube-proxy
-    cp -force /var/lib/kube-proxy/* /host/var/lib/kube-proxy
-    cp -force /var/run/secrets/kubernetes.io/serviceaccount/* /host/var/lib/kube-proxy/var/run/secrets/kubernetes.io/serviceaccount
-
-    wins cli process run --path /k/kube-proxy/kube-proxy.exe --args "--v=3 --config=/var/lib/kube-proxy/config.conf --proxy-mode=userspace --hostname-override=$env:NODE_NAME"
-
-kind: ConfigMap
-metadata:
-  labels:
-    app: kube-proxy
-  name: kube-proxy-windows
-  namespace: kube-system
-```
-
-For containerd runtime, replace the content of `run-script.ps1` with following:
-
-```yaml
-apiVersion: v1
-data:
-  run-script.ps1: |-
-    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
-    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/') 
-    New-Item -Path "c:/var/lib" -Name "kube-proxy" -ItemType "directory" -Force
-    ((Get-Content -path $mountPath/var/lib/kube-proxy/kubeconfig.conf -Raw) -replace '/var',"$($mountPath)/var") | Set-Content -Path /var/lib/kube-proxy/kubeconfig.conf
-    ((Get-Content -path /var/lib/kube-proxy/kubeconfig.conf -Raw) -replace '\/',"/") | Set-Content -Path /var/lib/kube-proxy/kubeconfig.conf
-    sed -i 's/mode: iptables/mode: \"\"/g' $mountPath/var/lib/kube-proxy/config.conf
-    & "$mountPath/k/kube-proxy/kube-proxy.exe" --config=$mountPath/var/lib/kube-proxy/config.conf --v=10 --proxy-mode=userspace --hostname-override=$env:NODE_NAME
-kind: ConfigMap
-metadata:
-  labels:
-    app: kube-proxy
-  name: kube-proxy-windows
-  namespace: kube-system
-```
-
-Set the `hostNetwork` option as true in spec of kube-proxy-windows daemonset.
-
-```yaml
-apiVersion: apps/v1
-kind: DaemonSet
-metadata:
-  labels:
-    k8s-app: kube-proxy
-  name: kube-proxy-windows
-  namespace: kube-system
-spec:
-  selector:
-    matchLabels:
-      k8s-app: kube-proxy-windows
-  template:
-    metadata:
-      labels:
-        k8s-app: kube-proxy-windows
-    spec:
-      hostNetwork: true
-```
+#### Add Windows antrea-agent DaemonSet
 
-For containerd runtime, Set `hostNetwork` as true and add options in spec and command.
+Starting from Antrea 1.13, you need to manually set the `kubeAPIServerOverride` field in the YAML configuration file as the proxy-all mode is enabled by default.
```

Comment:
Could we automate this step if we use Powershell for Windows installation anyway? @wenyingd 

Do not mean we need to do in this PR.

Reply:
We have added the command to modify the configuration in powershell script "Start-AntreaAgent.ps1" now, but the script is only working when running antrea-agent as Windows service. We now don't have automation to update in antrea-agent-windows-containerd.yml which is applied to run antrea-agent Pod on Windows. 

If we plan to add support for such automation, shall we assume users to run the script to on Windows or Linux? The CI testbed now applies the yml on master (Linux) Node.

Reply:
Ok, if we do not have script for containerd, we can keep it manual. Could we make a single command (with pipe) to download the yaml and change the parameter and even apply the yaml?

Reply:
I added a single command to apply yaml, does it look good to you? `curl -sL https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-windows-containerd.yml | sed "s|.*kubeAPIServerOverride: \"\"|    kubeAPIServerOverride: \"$(kubectl --kubeconfig=$KubeConfigFile config view -o jsonpath='{.clusters[0].cluster.server}')\"|g" | kubectl apply -f -`